Remotion/Core/Reflection/MethodInfoAdapter.cs

```diff
@@ -131,7 +132,7 @@ public object Invoke (object? instance, object?[]? parameters)
     {
       ArgumentUtility.CheckNotNull ("implementationType", implementationType);
 
-      if (!_methodInfo.DeclaringType.IsInterface)
+      if (!_methodInfo.DeclaringType!.IsInterface)
```

Comment:
Should we add a TODO for line 135 and 159?

Reply:
Yes, that would be another null bug.